README.md

```diff
@@ -1256,6 +1256,674 @@ The type of cost
+The main goal of this kind of simulation is to get a sens of the initial guesses passed to the solver.
+
+The second part of the example is to actually solve the program and then simulate the results from this solution.
+The main goal of this kind of simulation, especially in single shooting (that is not resetting the states at each node)
+is to validate the dynamics of multiple shooting. If they both are equal, it usually means that a great confidence
+can be held in the solution. Another goal would be to reload fast a previously saved optimized solution.
+
+### The pendulum.py file
+This is another way to present the pendulum example of the 'Getting started' section. 
+
+## Muscle driven OCP
+In this file, you will find four examples about muscle driven optimal control programs. The two first refer to traking 
+examples. The two last refer to reaching tasks. 
+
+### The muscle_activations_tracker.py file
+This is an example of muscle activation/skin marker or state tracking.
+Random data are created by generating a random set of muscle activations and then by generating the kinematics
+associated with these data. The solution is trivial since no noise is applied to the data. Still, it is a relevant
+example to show how to track data using a musculoskeletal model. In real situation, the muscle activation
+and kinematics would indeed be acquired via data acquisition devices.
+
+The difference between muscle activation and excitation is that the latter is the derivative of the former.
+
+The generate_data function is used to create random data. First, a random set of muscle activation is generated, as 
+shown below:
+`U = np.random.rand(n_shooting, n_mus).T`
+
+Then, the kinematics associated with these data are generated by integration, using `scipy.integrate.solve_ivp`. 
+
+To implement this tracking task, we use the ObjectiveFcn.Lagrange.TRACK_STATE objective function in the case of a state 
+tracking, or the `ObjectiveFcn.Lagrange.TRACK_MARKERS` objective function in the case of a marker tracking. We also use 
+the `ObjectiveFcn.Lagrange.TRACK_MUSCLES_CONTROL` objective function. The user can choose between marker or state 
+tracking thanks to the string `kin_data_to_track` which is one of the `prepare_ocp` function parameters. 
+
+### The muscle_excitations_tracker.py file
+This is an example of muscle excitation(EMG)/skin marker or state tracking.
+Random data are created by generating a random set of EMG and then by generating the kinematics associated with these
+data. The solution is trivial since no noise is applied to the data. Still, it is a relevant example to show how to
+track data using a musculoskeletal model. In real situation, the EMG and kinematics would indeed be acquired via
+data acquisition devices
+
+There is no huge difference with the precedent example. The difference between muscle activation and excitation is 
+that the latter is the derivative of the former.
```

Comment:
It is not the derivative per se, but there are dynamics equation that exists to link them

Reply:
Done